engine/src/components/tests/afterburner_tests.cpp

```diff
@@ -57,15 +57,15 @@ TEST(Afterburner, Sanity) {
     Afterburner afterburner;
     AfterburnerUpgrade upgrade(&afterburner);
 
-    afterburner.Load("", afterburner_string + upgrades_suffix_string);
+    afterburner.Load(afterburner_string + upgrades_suffix_string);
     upgrade.Load(upgrade_string + upgrades_suffix_string);
 
     EXPECT_EQ(afterburner.GetMass(), 0.0);
     EXPECT_EQ(afterburner.thrust.MaxValue(), 10.0);
     EXPECT_EQ(afterburner.speed.MaxValue(), 100.0);
     EXPECT_EQ(afterburner.GetConsumption(), 2.0);
 
-    EXPECT_EQ(upgrade.GetMass(), 5.0);
+    //EXPECT_EQ(upgrade.GetMass(), 5.0);
```

Comment:
Why are these being commented out? I don't understand

Reply:
If you look at Component.cpp, you'll see I've commented out the code to initialize name and mass of the upgrade. 
I have a general idea of where I want to go with this. Right now, the game uses cargo to represent upgrades and these (volume, price, name, etc.) are stored as part of the Master Part List (MPL).
Because these are part of Cargo and not Unit Asset, we need to process two separate elements. 
I'm not sure how to do that. I'm also not sure I need to. 
Right now, upgrade_view takes the key and pulls data directly from the Assets and MPL.

Reply:
So in other words, we're breaking functionality temporarily until we figure out how this should be done. I am not OK with that. At least not on `master`. The master branch is supposed to remain playable at all times, right?

Reply:
I agree here, the master should be playable at all times, so each of the issues identified should be fixed before this PR can be merged. 

Saying that, this is a step in the right direction, and so I'll be play testing this branch, and as the fixes come in for the various issues identified I will continue play testing until we are sure that this branch does not introduce any new issues into master.

Reply:
> So in other words, we're breaking functionality temporarily until we figure out how this should be done. I am not OK with that. At least not on master. The master branch is supposed to remain playable at all times, right?

Nope. Right now the game stores a list of `Cargo` items for each upgrade. I've made provisions in `Component` when I started development to store this information there. 

Commenting it out does not impact game play as this extra copy of Cargo attributes (mass, volume, name, description, etc.) is currently not used.

Reply:
Oh. Okay.